src/components/Sum.jsx

```diff
@@ -1 +1,3 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => (<p>{`Sum of ${a} and ${b} is ${a + b}`}</p>);
```

Comment:
One-line code is not the best solution for understandable reading of your code.

```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
);
```